### PR TITLE
feature: add `XDP_FLAGS*` in python lib

### PR DESCRIPTION
**Pull request by chenyuezhou**
resolve https://github.com/iovisor/bcc/issues/3445

add `XDP_FLAGS*` to `__init__.py`

I will add documentation in the following PR.

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: src/python/bcc/__init__.py [hunk 1]
```diff
@@ -171,6 +171,17 @@ class BPF(object):
...
 
+    # from xdp_flags uapi/linux/if_link.h
+    XDP_FLAGS_UPDATE_IF_NOEXIST = (1 << 0)
+    XDP_FLAGS_SKB_MODE = (1 << 1)
+    XDP_FLAGS_DRV_MODE = (1 << 2)
+    XDP_FLAGS_HW_MODE = (1 << 3)
+    XDP_FLAGS_REPLACE = (1 << 4)
+    XDP_FLAGS_MODES = (XDP_FLAGS_SKB_MODE | XDP_FLAGS_DRV_MODE | \
+                       XDP_FLAGS_HW_MODE)
+    XDP_FLAGS_MASK = (XDP_FLAGS_UPDATE_IF_NOEXIST | XDP_FLAGS_MODES | \
```
Comment (yonghong-song) [inline src/python/bcc/__init__.py:182]: XDP_FLAGS_MODES and XDP_FLAGS_MASK are mostly used by some library and kernel for sanity checks. bcc probably won't need this as user space knows exactly what to set. Could you remove them?

Comment (chenyuezhou) [reply src/python/bcc/__init__.py]: removed.

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Pull request merged**